code/modules/mining/shelters.dm

```diff
@@ -58,3 +58,18 @@
 	. = ..()
 	whitelisted_turfs = typecacheof(/turf/closed/mineral)
 	banned_objects = typecacheof(/obj/structure/stone_tile)
+
+/datum/map_template/shelter/charlie
+	name = "Shelter Charlie"
+	shelter_id = "shelter_charlie"
+	description = "A luxury elite bar which holds an entire bar \
```

Comment:
A luxury bar that holds an entire bar, what will they think of next?

Reply:
heh